R/R2R/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "R2R"
+version = v"1.0.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://sourceforge.net/projects/weinberg-r2r/files/R2R-1.0.6.tgz",
+                  "1ba8f51db92866ebe1aeb3c056f17489bceefe2f67c5c0bbdfbddc0eee17743d"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/R2R-*/
+
+# convert files to be patched to LF line endings (from CRLF)
+for f in src/SymbolicMath.cpp \
+         src/SymbolicMath.h \
+         src/GSCConsensus.cpp \
+         src/ParseOneStockholm.cpp \
+         src/PositionBackbone_MultiStemCircularSolver.cpp \
+         src/RnaDrawer.cpp;
+do
+    cp "$f" "$f".bak
+    tr -d '\015' < "$f".bak > "$f"
+done
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/pull/5133#issuecomment-1180886129.  I just realised you did the same for Rscape

Reply:
Ok, will rework the build to keep the line endings in the patches by setting .gitattributes appropriately here and in Rscape

Reply:
Thanks for the gitattributes tip!